resource_library.py

```diff
@@ -68,9 +74,9 @@ def generate_resource_and_src_list(base_dir, src_list):
     resource_list = generate_resource_and_src_list(base_dir, srcs)
 
     hfile = open(header_file, 'w')
-    print >> hfile, '#ifdef __cplusplus'
-    print >> hfile, 'extern "C" {'
-    print >> hfile, '#endif'
+    print('#ifdef __cplusplus', file=hfile)
```

Comment:
can we replace all those `print` calls? You can use `''' '''` to output multiple lines of text for better understanding.